finetune_adapter.py

```diff
@@ -43,14 +43,17 @@ def main(
     data_dir: Path = Path("data/alpaca"),
     checkpoint_dir: Path = Path("checkpoints/stabilityai/stablelm-base-alpha-3b"),
     out_dir: Path = Path("out/adapter/alpaca"),
+    accelerator = "cuda",
```

Comment:
I would suggest removing the accelerator entirely, as Fabric will choose it automatically if not set.

Reply:
Updated with this [commit](https://github.com/Lightning-AI/lit-parrot/pull/94/commits/37264f798a0b4cdc93414d74daaabb56991a313d)